arm_robots/scripts/tutorial/panda_basic_motion.py

```diff
@@ -1,11 +1,38 @@
 #! /usr/bin/env python
 
 import rospy
+import pdb
+import time
+import numpy as np
 from arm_robots.panda import Panda
 
+
 if __name__ == '__main__':
     rospy.init_node('panda_motion')
     panda = Panda()
     panda.connect()
-
+    pdb.set_trace()
+    panda.set_execute(False) # this will stop the robot from actually executing a path, good for testing
+    # panda.plan_to_position_cartesian(panda.nebula_arm, panda.nebula_wrist, target_position=[0.04, 0.2867, 1.17206])
```

Comment:
it's not a good idea to include commented out code. If this code works and is safe, consider commenting it back in. If it doesn't work or you don't want people to reference it, just delete it